resources/lang/fr/activity.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+return [
+    'login'                         => 'Connecté',
+    'registration'                  => 'enregistrement',
+    'view_home'                     => 'Affichage de la maison',
+    'view_searches'                 => 'Affichage des recherches',
+    'view_lost_people_details'      => 'Affichage des détails des personnes perdues',
+    'view_user_details'             => 'Affichage des détails de l utilisateur',
+    'view'                          => 'Visualisation',
+    'saved_new_info_about_search'   => 'Nouvelles informations enregistrées sur la recherche',
+    'clear_logs'                    => 'Effacer les journaux',
+    'time'                          => 'Temps',
+    'description'                   => 'La description',
+    'user'                          => 'l utilisateur',
```

Comment:
First letter has to be in uppercase. Also I think it should be just "Utilisateur" and not "L'utilisateur"